Assets/Scripts/PlayerMovement.cs

```diff
@@ -33,6 +32,8 @@ void Update()
         if (collision.gameObject.tag == "ground")
         {
             grounded = true;
+            remainingJumps = maxJumps; // if the player is grounded the remaningJumps variable is equal to the maxJum
+           
         }
     }
 
```

Comment:
You also don't need `OnCollisionExit2D` anymore.
As above, try and see why this is as such and try to understand the logic behind this